File: C++ Programming/Competitive Programming/Coding Challenges 8/program1.cpp
//Q1.Write a program which accept number from user and display its multiplication of factors.
//Input : 12
//Output : 144 (1 * 2 * 3 * 4 * 6)
//Input : 13
//Output : 1 (1)
//Input : 10
//Output : 10 (1 * 2 * 5)

#include <iostream>
using namespace std;

int MultFact(int iNo)
{
    int i = 1;
    int iMult = 1;

    if (iNo < 0)
    {
        iNo = -iNo;
    }

    while (i <= iNo / 2)
    {
        if (iNo / i * i == iNo)
        {
            iMult = iMult * i;
        }

        i++;
    }

    return iMult;
}
int main()
{
    int iValue = 0;
    int iRet = 0;

    cout << "Enter number ";
    cin >> iValue;

    iRet = MultFact(iValue);

    cout << "Output is: " << iRet;

    return 0;
}

File: Java Programming/Competitive Programming/Coding Challenges 13/program1.java
//Q1.Write a program which accept number from user and display below pattern.
// Input : 5
// Output : * * * * * # # # # #
// Input : 6
// Output : * * * * * * # # # # # # #
// Input : -5
// Output : * * * * * # # # # #
// Input : 2
// Output : * * # #

import java.util.Scanner;

class program1
{
    static void Display(int iNo)
    {
        if (iNo < 0)
        {
            iNo = -iNo;
        }

        String sStar = "*\t";
        String sHash = "#\t";

        System.out.print(sStar.repeat(iNo));
        System.out.print(sHash.repeat(iNo));
    }

    public static void main(String[] args)
    {
        Scanner sobj = new Scanner(System.in);

        int iValue = 0;

        System.out.print("Enter Number: ");
        iValue = sobj.nextInt();

        Display(iValue);

        sobj.close();
    }
}